rules/docker-iccpd.mk

```diff
@@ -0,0 +1,16 @@
+# docker image for iccpd agent
+
+DOCKER_ICCPD = docker-iccpd.gz
+$(DOCKER_ICCPD)_PATH = $(DOCKERS_PATH)/docker-iccpd
+$(DOCKER_ICCPD)_DEPENDS += $(SWSS) $(REDIS_TOOLS) $(ICCPD)
+$(DOCKER_ICCPD)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_STRETCH)
+SONIC_DOCKER_IMAGES += $(DOCKER_ICCPD)
+SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_ICCPD)
+
+SONIC_STRETCH_DOCKERS += $(DOCKER_ICCPD)
+
+$(DOCKER_ICCPD)_CONTAINER_NAME = iccpd
+$(DOCKER_ICCPD)_RUN_OPT += --net=host --privileged -t
```

Comment:
Remove --net=host. It is taken care generically in docker_image_ctl.j2 and not needed here. Will given run-time error.

Reply:
Thanks,done